Form-Controls/form-controls.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>T-shirt order</title>
+</head>
+<body>
+    <form>
+        <fieldset>
+            <legend>Customer Details</legend>
+
+            <label for="customer-name">Name</label>
+            <input type="text" name="name" id="customerName" minlength="2" placeholder="customer-name" required style="min-width:48px; min-height:48px;">>
```

Comment:
It looks there are duplicate `>` characters that are showing on the page

<img width="1796" height="468" alt="image" src="https://github.com/user-attachments/assets/02f78728-14ef-450e-adf4-e13ce124a154" />